src/config/timer.js

```diff
@@ -5,109 +5,115 @@ import { Message, MessageWithClickEvent } from "./message.js";
 export class Timer {
   #config;
   #chatComponent;
-	#timeComponent = new TimeComponent();
-	#lastMsgTime = (new Date()).getTime();
+  #timeComponent = new TimeComponent();
```

Comment:
TimeComponent uses Timer. Not the other way around. 
XxxComponent will use Timer no?
If you do it like this, then Timer class will have a lot of xxxComponent